src/raster/r.buildvrt.gdal/r.buildvrt.gdal.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
+"""
+ MODULE:       r.buildvrt.gdal
+ AUTHOR(S):    Stefan Blumentrath
+ PURPOSE:      Build GDAL Virtual Rasters (VRT) over GRASS GIS raster maps
+ COPYRIGHT:    (C) 2024 by stefan.blumentrath, and the GRASS Development Team
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+"""
+
+# %module
+# % description: Build GDAL Virtual Rasters (VRT) over GRASS GIS raster maps
+# % keyword: raster
+# % keyword: virtual
+# % keyword: gdal
+# % keyword: patch
+# %end
+
+# %option G_OPT_R_INPUTS
+# % key: input
+# % type: string
+# % required: no
+# % multiple: yes
+# %end
+
+# %option G_OPT_F_INPUT
+# % key: file
+# % required: no
+# %end
+
+# %option G_OPT_M_DIR
+# % key: vrt_directory
+# % description: Directory to store GDAL VRT files in. Default is: $GISDBASE/$PROJECT/$MAPSET/gdal
+# % required: no
+# %end
+
+# %option G_OPT_R_OUTPUT
+# %end
+
+# %flag
+# % key: m
+# % label: Read data range from metadata
+# % description: WARNING: metadata are sometimes approximations with wrong data range
+# %end
+
+# %flag
+# % key: r
+# % label: Create fast link without data range
+# % description: WARNING: some modules do not work correctly without known data range
+# %end
+
+# %rules
+# % required: input,file
+# %end
+
+
+import sys
+
+from pathlib import Path
+
+import grass.script as gs
+
+
+def get_raster_gdalpath(map_name, check_linked=True, gis_env=None):
+    """Get get the path to a raster map that can be opened by GDAL
+
+    Checks for GDAL source of linked raster data and returns those
+    if not otherwise requested or the path to the header of
+    raster maps in native GRASS GIS format"""
+    if check_linked:
+        # Check GDAL link header
+        map_info = gs.find_file(map_name)
+        header_path = (
+            Path(gis_env["GISDBASE"])
+            / gis_env["LOCATION_NAME"]
+            / map_info["mapset"]
+            / "cell_misc"
+            / map_info["name"]
+            / "gdal"
+        )
+        if header_path.exists():
+            gdal_path = Path(
+                gs.parse_key_val(header_path.read_text().replace(": ", "="))["file"]
```

Comment:
Would the following work?

```suggestion
                gs.parse_key_val(header_path.read_text(), sep=": "))["file"]
```